pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -342,41 +342,41 @@ const (
 	// ShootOperationRetry is a constant for an annotation on a Shoot indicating that a failed Shoot reconciliation shall be
 	// retried.
 	ShootOperationRetry = "retry"
-	// ShootOperationRotateCredentialsStart is a constant for an annotation on a Shoot indicating that the rotation of
+	// OperationRotateCredentialsStart is a constant for an annotation on a Shoot indicating that the rotation of
```

Comment:
Similar for other occurrences
```suggestion
        // OperationRotateCredentialsStart is a constant for an annotation indicating that the rotation of
```